apps/ledger-live-desktop/src/renderer/reducers/settings.ts

```diff
@@ -143,7 +144,7 @@ export const getInitialLanguageAndLocale = (): { language: Language; locale: Loc
   const systemLocal = getAppLocale();
 
   // Find language from system locale (i.e., en, fr, es ...)
-  const languageId = LanguageIds.find(lang => systemLocal.startsWith(lang));
+  const languageId = LanguageIdsNotFeatureFlagged.find(lang => systemLocal.startsWith(lang));
```

Comment:
can't be thai?

Reply:
It can't be thai because at this point we can't know if the feature flag is enabled or not. It's a limitation of our current implementation that we can improve later on